django_netjsonconfig/static/django-netjsonconfig/js/widget.js

```diff
@@ -226,16 +228,16 @@
         // otherwise load immediately
         bindLoadUi();
     });
-})(django.jQuery);
+}(django.jQuery));
 
 var matchKey = (function () {
     var elem = document.documentElement;
-    if (elem.matches) return 'matches';
-    else if (elem.webkitMatchesSelector) return 'webkitMatchesSelector';
-    else if (elem.mozMatchesSelector) return 'mozMatchesSelector';
-    else if (elem.msMatchesSelector) return 'msMatchesSelector';
-    else if (elem.oMatchesSelector) return 'oMatchesSelector';
-})();
+    if (elem.matches) { return 'matches'; }
+    if (elem.webkitMatchesSelector) { return 'webkitMatchesSelector'; }
```

Comment:
I don't think changing all the `else if` to `if` is a good idea

Reply:
In this case, when the `if` matches, it returns so the rest of the `if`s are never reached. It's the same as using an `else if`, but without jslint complaining 😛

Reply:
correct!